src/components/List.js

```diff
@@ -0,0 +1,27 @@
+import React, {PropTypes} from 'react'
+import CSSModules from 'react-css-modules'
+import styles from './List.css'
+
+const listContainerStyle = {
+  height: (window.innerHeight - 200) + 'px'
+}
+class List extends React.Component {
+  componentWillMout () {
```

Comment:
this is to make the navigation expand all the way to the bottom, was there from the last version but I moved it to the List component, I guess I can use CSS to do that also